src/averageAges.js

```diff
@@ -1,5 +1,10 @@
 'use strict';
 
+function calculateAverageAge(arr) {
```

Comment:
don't use data type names for variable names
let's create more understandable variable name instead of `arr`